lib/signaling/v2/room.js

```diff
@@ -516,6 +526,17 @@ function handleSubscriptionFailures(room) {
       remoteTrack.subscribeFailed(createTwilioError(error.code, error.message));
     }
   });
+
+  const subscribedTrackIds = new Set(Array.from(room._subscribed.keys()));
+  remoteTracks.forEach(remoteTrack => {
+    if (!subscribedTrackIds.has(remoteTrack.id)) {
+      remoteTrack.setTrackTransceiver(null);
+      return;
+    }
+    if (!remoteTrack.isSubscribed) {
```

Comment:
Should we have a clause that checks if the `remoteTrack`'s SID is in `room._subscribed`? i.e., we do the `_getTrackReceiver` call _unless_ we know we should be subscribed?

Reply:
@markandrus I think this is implied if the first `if` statement (above this one) is false.